### PR TITLE
#OB-553 fix: Analytics service issue fixes for OCI

### PR DESCRIPTION
**Pull request by sowmya-dixit**
<table>
<td>
  <a href="https://project-sunbird.atlassian.net/browse/OB-553" title="OB-553" target="_blank">RFCOB-553</a>  Support OCI deployment
  </td></table>
  <br />
 

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Enhancement (additive changes to improve performance)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: kubernetes/helm_charts/core/analytics/templates/deployment.yaml
Comment (santhosh-tg) [inline kubernetes/helm_charts/core/analytics/templates/deployment.yaml:1]: @sowmya-dixit  please replace the variables that are referencing to azure with the general cloud variables. for ex: azure_private_account_name can be replaced with cloud_private_storage_accountname

File: ansible/roles/stack-sunbird/templates/analytics_api_service.conf
Comment (santhosh-tg) [inline ansible/roles/stack-sunbird/templates/analytics_api_service.conf:1]: @sowmya-dixit  should this not be an ansible variable  , how does the value get substituted here?

File: kubernetes/helm_charts/core/analytics/templates/deployment.yaml
Comment (sowmya-dixit) [reply kubernetes/helm_charts/core/analytics/templates/deployment.yaml:1]: Done

File: ansible/roles/stack-sunbird/templates/analytics_api_service.conf
Comment (sowmya-dixit) [reply ansible/roles/stack-sunbird/templates/analytics_api_service.conf:1]: These will be substituted from ENV variables.

**Review by santhosh-tg — APPROVED** [review]

**Pull request merged**